Resources/views/CRUD/list__select.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends admin.getTemplate('base_list_field') %}
 
 {% block field %}
-    <a class="btn btn-default" href="{{ admin.generateObjectUrl('edit', object) }}">
+    <a class="btn btn-default" href="{{ admin.generateObjectUrl('list', object) }}">
```

Comment:
Why are you passing the object to the route? A list route doesn't need this @sergiostrampelli